sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/TicketCache.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.appcenter.utils;
+
+import android.support.annotation.VisibleForTesting;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TicketCache {
```

Comment:
This seems very generic, a cache of a string-string mapping... Could we need this in other areas in the future?  Should it be refactored as a generic cache?  Just a thought.

Reply:
It should contain only tickets, nothing else to avoid colliding.